nameko/standalone/events.py

```diff
@@ -31,6 +34,11 @@ def event_dispatcher(nameko_config, **kwargs):
     def dispatch(service_name, event_type, event_data):
         conn = Connection(nameko_config[AMQP_URI_CONFIG_KEY])
 
+        if KOMBU_SERIALIZER_CONFIG_KEY in nameko_config.keys():
+            serializer = nameko_config[KOMBU_SERIALIZER_CONFIG_KEY]
+        else:
+            serializer = DEFAULT_KOMBU_SERIALIZER
```

Comment:
i think we should just drop the `foo.get(key) or default` pattern, as the behaviour is somewhat unexpected (certain config values just being ignored)

i did some digging for though the git history of this for the `max_workers`, and i can't see any direct reasons to override e.g. an explicit `None` or `0`. It won't work of course, but neither will setting the value to `foo`. We may add some validation/sanity checking at some point later, but for now i think it's fine to "allow" those config values and just let any errors bubble